src/components/vanilla/Container.tsx

```diff
@@ -81,24 +83,17 @@ export default ({
   useFont();
 
   return (
-    <div className="h-full relative font-embeddable text-sm flex flex-col">
-      {props.enableDownloadAsCSV ? (
-        <div className={`${!props.title ? 'h-[40px] w-full' : ''}`}>
```

Comment:
I'll fix it - we had to move the icons down below the chart, in the code, due to z-index issues preventing the menu buttons from being clicked on because the canvas was taking precedence over them. I can put in a spacer